exec/java-exec/src/main/java/org/apache/drill/exec/client/DrillClient.java

```diff
@@ -98,6 +100,8 @@
  * String into ByteBuf.
  */
 public class DrillClient implements Closeable, ConnectionThrottle {
+  public static final String DEFAULT_CLIENT_NAME = "Apache Drill Java client";
```

Comment:
Also, since I already have a big C++ patch in progress, the C++ change will be done in that branch